platform-includes/getting-started-config/javascript.node.mdx

```diff
@@ -11,10 +11,6 @@ Sentry.init({
   // We recommend adjusting this value in production
   tracesSampleRate: 1.0,
 });
-
-// http and other libraries are only instrumented
-// if required _after_ Sentry has been initialized
-const http = require("http");
 ```
 
 ```javascript {tabTitle:ESM}
```

Comment:
Adding a filename here now would be nice!

Reply:
Fixed